src/esp/gfx/CubeMap.cpp

```diff
@@ -403,6 +403,10 @@ bool CubeMap::saveTexture(TextureType type,
   if (!(converter = manager.loadAndInstantiate("AnyImageConverter"))) {
     return false;
   }
+  // set image converter flags if gfx logging is quieted
+  if (!isLevelEnabled(logging::Subsystem::gfx, logging::LoggingLevel::Debug)) {
```

Comment:
Hmm, doesn't this mean that if I set "warnings only", i.e. [LoggingLevel::Warning](https://aihabitat.org/docs/habitat-sim/namespaceesp_1_1logging.html#af964da7d81bdebf67382b6f9dc78c4d1a0eaadb4fcb48a0a0ed7bc9868be9fbaa), warnings from the importer will be suppressed?

As in, shouldn't that be checking the level between `Debug` and `Quiet` instead? Apologies if I misunderstood how the levels work.

```suggestion
  if (!isLevelEnabled(logging::Subsystem::gfx, logging::LoggingLevel::Warning)) {
```

Reply:
Makes sense. Moved this to Warning level.